src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -160,9 +164,19 @@ private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appCo
                 await sessionManager.SetAsync(feature, enabled).ConfigureAwait(false);
             }
 
+            sendImpression(feature, enabled);
+
             return enabled;
         }
 
+        private void sendImpression(string feature, bool enabled)
```

Comment:
Naming convention
_Consider the other comment to change it to async then the signature would be slightly different._
```suggestion
        private void SendImpression(string feature, bool enabled)
```